File: Top 100 Codes/04-Important Codes related to Arrays/18-Counting the number of even and odd elements in an array/Countingthenumberofevenandoddelementsinanarray.cpp
#include <iostream>
using namespace std;

int main() {
    int arr[] = {12, 7, 9, 20, 33, 40, 55, 68}; //  array
    int n = sizeof(arr) / sizeof(arr[0]); // Calculate array size
    int evenCount = 0, oddCount = 0;

    for (int i = 0; i < n; i++) {
        if (arr[i] % 2 == 0)
            evenCount++;
        else
            oddCount++;
    }

    cout << "Number of even elements: " << evenCount << endl;
    cout << "Number of odd elements: " << oddCount << endl;

    return 0;
}

File: Top 100 Codes/04-Important Codes related to Arrays/18-Counting the number of even and odd elements in an array/Countingthenumberofevenandoddelementsinanarray.py
# Predefined array
arr = [12, 7, 9, 20, 33, 40, 55, 68]

# Count even and odd numbers
even_count = sum(1 for num in arr if num % 2 == 0)
odd_count = len(arr) - even_count

# Print results
print("Number of even elements:", even_count)
print("Number of odd elements:", odd_count)
